activerecord/lib/active_record/relation.rb

```diff
@@ -641,6 +641,13 @@ def inspect
       "#<#{self.class.name} [#{entries.join(', ')}]>"
     end
 
+    protected
+
+      def load_records(records)
```

Comment:
@sgrif BTW I am sorry if force-pushing made the review difficult. Let me know if you prefer separate commits from now on. I will squash them later again if necessary.